pkg/converter/struct_v1.go

```diff
@@ -0,0 +1,113 @@
+package converter
+
+import (
+	"fmt"
+
+	upiv1 "github.com/caraml-dev/universal-prediction-interface/gen/go/grpc/caraml/upi/v1"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+const (
+	IntType     = "INT64"
+	NumberType  = "FLOAT64"
+	StringType  = "STRING"
```

Comment:
for my understanding where does `INT64 FLOAT64 STRING` come from? 

Also was confused abit when I saw `NumberType` but guess its not outputted anywhere

Reply:
> for my understanding where does INT64 FLOAT64 STRING come from?

This is coming from the BQ column type. It'll be used later on during query time from the log table. I think it's better to rename the const to reflect its value better.

Reply:
ah, I thought the infer schema will be FLOAT (happen to have a ss below), number is only used in structpb I think.

Was just a curious question if there was something that I missed, thanks!

Reply:
Oh you are right. Let me update it accordingly .